tasklist/qa/integration-tests/src/test/java/io/camunda/tasklist/util/ElasticsearchTestExtension.java

```diff
@@ -53,9 +54,11 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Scope;
 import org.springframework.stereotype.Component;
 
 @Component
+@Scope(SCOPE_PROTOTYPE)
```

Comment:
For ElasticSearch, we are using different index prefixes per test, to avoid mutual impacts between tests.
I have noticed that the tests in the same Test class are using the same prefix, which sometimes give strange failures . This is a "regression" that we introduced when we migrated from jUnit4 to jUnit5 (unrelated to the changes here, but should make the tests more reliable)
`@Scope(SCOPE_PROTOTYPE)` should create a new `ElasticsearchTestExtension` bean with fresh index prefix rather using `ElasticsearchTestExtension` as singleton bean

Reply:
You can avoid the creation of a new instance for every single bean injection, unless absolutely necessary for
test isolation using 

`@Scope(SCOPE_SINGLETON)`

When we define a bean with the singleton scope, the container creates a single instance of that bean; all requests for that bean name will return the same object, which is cached.

Reply:
I think this is really necessary for test stability, I have noticed that one test fails (for real reason) it causes the failure of other tests (false positives). Making sure that each test uses its index namespace will reduce the number of false positives